sys-devel/llvm/llvm-3.4.2-r100.ebuild

```diff
@@ -203,8 +197,8 @@ src_install() {
 	if ! use clang; then
 		rm "${WORKDIR}"/${PN}-3.4-manpages/clang.1 || die
 	else
-		for tool in clang{,++}{,-${PV}} ; do
-			dosym /usr/bin/${tool} /usr/bin/${CHOST}-${tool}
+		for tool in clang{,++} ; do
+			dosym ${tool} /usr/bin/${CHOST}-${tool}
```

Comment:
${tool} doesn't include path, and I don't see a cd or pushd, so does this work as anticipated?

I guess you wanted to use something like:
`dosym ${EPREFIX}/usr/bin/${tool} /usr/bin/${CHOST}-${tool}`
(as per bug #615594)?